autoinstrumentation/python/requirements.txt

```diff
@@ -1,61 +1,61 @@
-opentelemetry-distro==0.48b0
+opentelemetry-distro==0.50b0
 # We don't use the distro[otlp] option which automatically includes exporters since gRPC is not appropriate for
 # injected auto-instrumentation, where it has a strict dependency on the OS / Python version the artifact is built for.
 opentelemetry-exporter-otlp-proto-http==1.27.0
-opentelemetry-exporter-prometheus==0.48b0
```

Comment:
I'm not sure I like doing this in a release PR.

Reply:
yeah it's decently out to date... i can leave it out though if you prefer.

Reply:
I would prefer to be done this in a separate PR as well

Reply:
It should be done in a separate PR that will publish the python image and then use that image in this PR.